patches/BUILD

```diff
@@ -0,0 +1 @@
+licenses(["notice"])
```

Comment:
No idea what this does.

Reply:
Same, no idea what it does. Removed. (But I had to keep the BUILD file)